KnoraBuild.sbt

```diff
@@ -335,12 +335,12 @@ lazy val webapi = knoraModule("webapi")
 
             fork := true, // always fork
 
-            run / javaOptions := webapiJavaRunOptions,
+            // run / javaOptions := webapiJavaRunOptions,
```

Comment:
Why are you removing the java option?

Reply:
You said I should comment out the definition of `webapiJavaRunOptions`, so I also have to comment out all references to it, otherwise the file doesn’t compile. Or have I misunderstood?

Reply:
Ah, sorry. What I meant is to comment out https://github.com/dhlab-basel/Knora/blob/a3c5167156fcdded96f97a7037ead7e0f20240fe/KnoraBuild.sbt#L444-L447

This should have the same effect.

Reply:
We need the other options.

Reply:
OK, sorry about the misunderstanding. How's this?